src/zope/app/locales/extract.py

```
@@ -205,9 +205,14 @@ def __repr__(self):
 class POTMaker(object):
     """This class inserts sets of strings into a POT file."""
 
-    def __init__(self, output_fn, path):
+    def __init__(self, output_fn, path, header_template=None):
         self._output_filename = output_fn
         self.path = path
+        if header_template is not None and os.path.exists(header_template):
+            with open(header_template, "r") as file:
+                self._pot_header = file.read()
```

Why would you want to ignore a specified header_template argument when the file does not exist, instead of reporting the error?

---

It was in the code ported from `z3c.recipe.i18n`. But you are right, we could raise an exception if the path does not exist.